css/style.css

```diff
@@ -541,3 +566,9 @@ body {
     margin: 64px auto 0;
   }
 }
+
+@media (max-width: 287px) {
+  .test__body {
+    font-size: 20px;
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5985492)